keyboards/anne_pro/config.h

```diff
@@ -0,0 +1,84 @@
+/* Copyright 2019 Michiel Visser
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x0483
+#define PRODUCT_ID      0x5710
```

Comment:
This is STM's device ID, and for Joystick in FS Mode. 
https://www.the-sz.com/products/usbid/index.php?v=0x0483&p=0x5710&n=

This should/needs to be changed.

Reply:
I am unsure what to do with this, since the original firmware for this keyboard uses this VID/PID pair. This is how the original firmware enumerates on my machine:

```
obins anne keyboard :

  Product ID:	0x5710
  Vendor ID:	0x0483  (STMicroelectronics)
  Version:	2.00
  Serial Number:	STM32
  Speed:	Up to 12 Mb/s
  Manufacturer:	STMicroelectronics
  Location ID:	0x14110000 / 17
  Current Available (mA):	500
  Current Required (mA):	400
  Extra Operating Current (mA):	0
```

Reply:
Well, using 0xFEED for the VID, and a something else for the PID would probably be best in the meanwhile. 

But I'll defer to input from other @qmk/collaborators

Reply:
```suggestion
#define VENDOR_ID       0xFEED
#define PRODUCT_ID      0xFBF1
```